src/app/backend-api.service.ts

```diff
@@ -2613,298 +1761,128 @@ export class BackendApiService {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathAdminGetAllReferralInfoForUser, AdminPublicKey, {
-      UserPublicKeyBase58Check,
-      Username,
-      AdminPublicKey,
-    });
-  }
-
-  AdminDownloadReferralCSV(endpoint: string, AdminPublicKey: string): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathAdminDownloadReferralCSV, AdminPublicKey, {
-      AdminPublicKey,
-    });
-  }
-
-  AdminDownloadRefereeCSV(endpoint: string, AdminPublicKey: string): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathAdminDownloadRefereeCSV, AdminPublicKey, {
-      AdminPublicKey,
-    });
-  }
-
-  AdminUploadReferralCSV(endpoint: string, AdminPublicKey: string, CSVRows: Array<Array<String>>): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathAdminUploadReferralCSV, AdminPublicKey, {
-      AdminPublicKey,
-      CSVRows,
-    });
-  }
-
-  GetReferralInfoForUser(endpoint: string, PublicKeyBase58Check: string): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathGetReferralInfoForUser, PublicKeyBase58Check, {
-      PublicKeyBase58Check,
-    });
-  }
-
-  GetReferralInfoForReferralHash(
-    endpoint: string,
-    ReferralHash: string
-  ): Observable<{ ReferralInfoResponse: any; CountrySignUpBonus: CountryLevelSignUpBonus }> {
-    return this.post(endpoint, BackendRoutes.RoutePathGetReferralInfoForReferralHash, {
-      ReferralHash,
-    });
-  }
-
-  AdminResetTutorialStatus(endpoint: string, AdminPublicKey: string, PublicKeyBase58Check: string): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathAdminResetTutorialStatus, AdminPublicKey, {
-      PublicKeyBase58Check,
-      AdminPublicKey,
-    });
-  }
-
-  AdminUpdateTutorialCreators(
-    endpoint: string,
-    AdminPublicKey: string,
-    PublicKeyBase58Check: string,
-    IsRemoval: boolean,
-    IsWellKnown: boolean
-  ): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathAdminUpdateTutorialCreators, AdminPublicKey, {
-      PublicKeyBase58Check,
-      IsRemoval,
-      IsWellKnown,
-      AdminPublicKey,
-    });
+    return from(
+      adminUpdateHotFeedUserMultiplier({
+        Username,
+        InteractionMultiplier,
+        PostsMultiplier,
+      })
+    );
   }
 
-  GetTutorialCreators(endpoint: string, PublicKeyBase58Check: string, ResponseLimit: number): Observable<any> {
-    return this.post(endpoint, BackendRoutes.RoutePathGetTutorialCreators, {
-      ResponseLimit,
-      PublicKeyBase58Check,
-    });
+  AdminGetHotFeedUserMultiplier(Username: string): Observable<any> {
+    return from(
+      adminGetHotFeedUserMultiplier({
+        Username,
+      })
+    );
   }
 
+  /**
+   * This is for the legacy tutorial thing. I don't *think* we can remove it
+   * since I think we still need it to set the users tutorial status to
+   * complete. I forget whether the backend already does this or not.
+   */
```

Comment:
to clarify

Reply:
Let's leave this for now - tutorial status is still used throughout the backend